data/aecore/.genesis/accounts_test.json

```diff
@@ -1,3 +1,3 @@
 {
-   "ak_2a1j2Mk9YSmC1gioUq4PWRm3bsv887MbuRVwyv4KaUGoR1eiKi": 100000000000001
+   "ak_2a1j2Mk9YSmC1gioUq4PWRm3bsv887MbuRVwyv4KaUGoR1eiKi": 100000000000000000000001
```

Comment:
In this version of the code i.e. without 9efcab30 this impacts testnet too.
This PR may have some not explicit conflicts with 9efcab30

Reply:
I see the rebase included the commit that copied this file with other name for testnet, so this is resolved.